packages/go_router/test/go_router_test.dart

```diff
@@ -408,14 +427,18 @@ void main() {
         ),
       ];
 
-      final GoRouter router = _router(routes);
-      router.go('/', extra: 1);
```

Comment:
Was the `/` test intentionally removed? Why is that?

Reply:
Not sure, I'll add it back.